packages-node/providence-analytics/src/program/utils/optimised-glob.js

```diff
@@ -315,9 +317,12 @@ export async function optimisedGlob(globOrGlobs, providedOptions = {}) {
   const ignoreGlobs = options.ignore.map((/** @type {string} */ g) =>
     g.startsWith('!') ? g : `!${g}`,
   );
+  const globs = toUniqueArray([...regularGlobs, ...ignoreGlobs]);
 
   const optionsNotSupportedByNativeGlob = ['onlyDirectories', 'dot'];
-  const doesConfigAllowNative = !optionsNotSupportedByNativeGlob.some(opt => options[opt]);
+  const hasGlobWithFullPath = globs.some(g => g.startsWith('/'));
```

Comment:
Good one, I will check what globby does

Reply:
Seems like globby doesn't filter out these, optimisedGlob does (which is correct behavior imho). I will leave a comment addressing thism and for now leave it at this (since it's an edge case within an edge case with an easy workaround :))